commit a0be15830ef808763043a9137a1a8d5da6f4b804
Author: Kirill A. Korinsky <kirill@korins.ky>
Date:   Sat Sep 16 00:56:33 2023 +0200

    Add `--with-python=PATH` to enforce used version of python

diff --git configure.ac configure.ac
index d72d10dc5..a1bb7f373 100644
--- configure.ac
+++ configure.ac
@@ -88,7 +88,11 @@ AC_CHECK_PROGS([DOT], [dot])
 _VARNISH_VMOD_LDFLAGS
 
 # Check for python.
-_VARNISH_CHECK_PYTHON
+AC_ARG_WITH([python],
+  AS_HELP_STRING([--with-python=PATH], [Location of python (auto)]),
+  [PYTHON="$withval"],
+  [_VARNISH_CHECK_PYTHON])
+
 
 # Check for libraries.
 _VARNISH_SEARCH_LIBS(pthread, pthread_create, [thr pthread c_r])
